### PR TITLE
Fix using REGISTRY instead of registry for PUSH_MANIFEST_IMAGES

### PR DESCRIPTION
**Pull request by Brian-McM**

### CODE REVIEW

File: Makefile.common [hunk 1]
```diff
@@ -107,7 +107,7 @@ DEV_REGISTRY ?= $(firstword $(DEV_REGISTRIES))
 NONMANIFEST_REGISTRIES      ?= quay.io
 MANIFEST_REGISTRIES         ?= $(DEV_REGISTRIES:$(NONMANIFEST_REGISTRIES)%=)
 
-PUSH_MANIFEST_IMAGES := $(foreach registry,$(MANIFEST_REGISTRIES),$(foreach image,$(BUILD_IMAGES),$(call filter-registry,$(REGISTRY))$(image)))
+PUSH_MANIFEST_IMAGES := $(foreach registry,$(MANIFEST_REGISTRIES),$(foreach image,$(BUILD_IMAGES),$(call filter-registry,$(registry))$(image)))
```
Comment (stevegaossou) [inline Makefile.common:110]: It looks like every other reference to that variable is in all-caps though.

[Here](https://github.com/projectcalico/go-build/blob/e53d5a2516a25dfbdc87f25aa512f91e11ef1212/Makefile.common#L732-L739) and [here](https://github.com/projectcalico/go-build/blob/e53d5a2516a25dfbdc87f25aa512f91e11ef1212/Makefile.common#L759-L766).

Comment (Brian-McM) [reply Makefile.common:110]: It's a different variable, it's a for loop so this is referencing registry in `$(foreach registry,$(MANIFEST_REGISTRIES)`

**Review by stevegaossou — APPROVED** [review]
LGTM

**Review by mgleung — APPROVED** [review]
LGTM

**Pull request merged**